client/ayon_core/hosts/zbrush/hooks/create_zmenu_script.py

```diff
@@ -0,0 +1,88 @@
+"""Pre-launch to force zbrush startup script."""
+import os
+import subprocess
+from ayon_core.hosts.zbrush import ZBRUSH_HOST_DIR
+from ayon_core.lib import get_ayon_launcher_args
+from ayon_core.lib.applications import PreLaunchHook, LaunchTypes
+
+
+
+class CreateZMenuScript(PreLaunchHook):
+    """Create AYON Menu Zscript to Zbrush.
+
+    Note that this works in combination whit Zbrush startup script
+    to successfully install zscripts.menu
+
+    Hook `GlobalHostDataHook` must be executed before this hook.
+    """
+    app_groups = {"zbrush"}
+    order = 12
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+
+        zscript_path = os.path.join(ZBRUSH_HOST_DIR, "api", "zscripts")
+        os.makedirs(zscript_path, exist_ok=True)
+        zscript_txt = os.path.join(zscript_path, "ayon_zbrush_menu.txt")
+        with open(zscript_txt, "w") as zscript:
+            zscript.write(self.ayon_menu())
+            zscript.close()
+        self.launch_context.env["AYON_ZBRUSH_CMD"] = os.environ["AYON_ZBRUSH_CMD"]
+    def ayon_menu(self):
+        zclient_path = os.path.join(ZBRUSH_HOST_DIR, "api", "widgets.py")
+        zclient_path = zclient_path.replace("\\", "/")
+        os.environ["AYON_ZBRUSH_CMD"] = subprocess.list2cmdline(
+            get_ayon_launcher_args()
```

Comment:
Please check you linter

```suggestion

        zscript_path = os.path.join(ZBRUSH_HOST_DIR, "api", "zscripts")
        os.makedirs(zscript_path, exist_ok=True)
        zscript_txt = os.path.join(zscript_path, "ayon_zbrush_menu.txt")
        with open(zscript_txt, "w") as zscript:
            zscript.write(self.ayon_menu())
            zscript.close()
        self.launch_context.env["AYON_ZBRUSH_CMD"] = os.environ["AYON_ZBRUSH_CMD"]

    def ayon_menu(self):
        zclient_path = os.path.join(ZBRUSH_HOST_DIR, "api", "widgets.py")
        zclient_path = zclient_path.replace("\\", "/")
        os.environ["AYON_ZBRUSH_CMD"] = subprocess.list2cmdline(
            get_ayon_launcher_args()
```

@jakubjezek001 can we get the hound active? I feel like there might be quite some PRs sneaking in invalid cosmetics currently?